docs/docs/rest-api/public/api/v2/apps.raml

```diff
@@ -0,0 +1,451 @@
+#
+put:
+  description:
+    Change multiple applications either by upgrading existing ones or creating new ones.
+    If there is an update to an already running application, the application gets upgraded.
+    All instances of this application get replaced by the new version.
+    The order of dependencies will be applied correctly.
+    The upgradeStrategy defines the behaviour of the upgrade.
+
+    If the id of the application is not known, the application gets started.
+    The order of dependencies will be applied correctly.
+    It is possible to mix upgrades and installs.
+
+    If you have more complex scenarios with upgrades, use the groups endpoint.
+  is: [ secured ]
+  body:
+    application/json:
+      example: !include examples/apps_create.json
+  responses:
+    200:
+      description:
+        A deployment is started which has a unique deployment identifier.
+        The related deployment can be fetched from the /v2/deployments endpoint.
+        If the deployement is gone from the list of deployments, than this means it is finished.
+        As long as the deployment runs, the effect of that change operation is visible only partially.
+      body:
+        application/json:
+          example: !include examples/deployments_result.json
+    409:
+      description: There is an already deployed application with this name
+      body:
+        application/json:
+          example: |
+            {"message":"An app with id [/existing_app] already exists."}
+    400:
+      description: The application definition provided in the body is not valid.
+      body:
+        application/json:
+          example: |
+            {"message":"Invalid JSON","details":[{"path":"/id","errors":["error.expected.jsstring"]}]}
+    422:
+      description: The entity send can not be preocessed, since there are validation errors
+      body:
+        application/json:
+          example: |
+            {
+              "message": "Bean is not valid",
+              "errors": [
+                {
+                  "attribute": "upgradeStrategy.minimumHealthCapacity",
+                  "error": "is greater than 1"
+                }
+              ]
+            }
+
+
+
+
+###
+# Create application
+#
+post:
+  description: Create and start a new application.
+  is: [ secured ]
+  body:
+    application/json:
+      example: !include examples/app.json
+      schema: AppDefinition
+  responses:
+    201:
+      description: The application has been created and a deployment is started.
+      body:
+        application/json:
+          example: !include examples/app.json
+    409:
+      description: There is an already deployed application with this name
+      body:
+        application/json:
+          example: |
+            {"message":"An app with id [/existing_app] already exists."}
+    400:
+      description: The application definition provided in the body is not valid.
+      body:
+        application/json:
+          example: |
```

Comment:
Maybe the example should be short and only show a basic example – and the schema open by default? If the example is long, the schema-button is a long way down the page.